File: examples/src/Slides/Notes/RemoveNotesAtSpecificSlide.cs
﻿using Aspose.slides.Export

/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Slides for .NET API reference 
when the project is build. Please check https:#docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Slides for .NET API from http:#www.aspose.com/downloads, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using http:#www.aspose.com/community/forums/default.aspx
*/

namespace Aspose.slides.Examples.CSharp.slides.Notes
{
    class RemoveNotesAtSpecificSlide
    {
        public static void Run()
        {
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Slides_Presentations_Notes()

            #ExStart:RemoveNotesAtSpecificSlide
            # Instantiate a Presentation object that represents a presentation file 
            Presentation presentation = new Presentation(dataDir + "AccessSlides.pptx")

            # Removing notes of first slide
            INotesSlideManager mgr = presentation.slides[0].NotesSlideManager
            mgr.RemoveNotesSlide()

            # Save presentation to disk
            presentation.save(dataDir + "RemoveNotesAtSpecificSlide_out.pptx", slides.export.SaveFormat.PPTX)

            #ExEnd:RemoveNotesAtSpecificSlide

        }
    }
}

File: examples/src/slides/notes/remove_notes_at_specific_slide.py
import aspose.slides as slides


def remove_notes_at_specific_slide(global_opts):
    # Instantiate a Presentation object that represents a presentation file
    with slides.Presentation(global_opts.data_dir + "welcome-to-powerpoint.pptx") as presentation:
        # Removing notes of first slide
        notes_slide_manager = presentation.slides[0].notes_slide_manager
        notes_slide_manager.remove_notes_slide()

        # Save presentation to disk
        presentation.save(global_opts.out_dir + "notes_remove_notes_slide_out.pptx", slides.export.SaveFormat.PPTX)
